dask/base.py

```diff
@@ -765,7 +774,34 @@ def label(x):
     elif color:
         raise NotImplementedError("Unknown value color=%s" % color)
 
-    return dot_graph(dsk, filename=filename, **kwargs)
+    # Determine which engine to dispatch to, first checking the kwarg, then config,
+    # then whichever of graphviz or ipycytoscape are installed, in that order.
+    engine = engine or config.get("visualization.engine", None)
+
+    if not engine:
+        try:
+            import graphviz  # noqa: F401
+
+            engine = "graphviz"
+        except ImportError:
+            pass
+        try:
+            import ipycytoscape  # noqa: F401
+
+            engine = "cytoscape"
+        except ImportError:
+            pass
```

Comment:
From what I can tell, since we're setting `visualization.engine = "graphviz"` by default, we'll never run this section (by default). My guess is we still want to set `engine="cytoscape"` under default settings if we don't have `graphviz` installed, but do have `ipycytoscape` installed

Reply:
Good catch, thanks. I'll set `visualization.engine: null` in the default settings.